packs/bestiary-effects/effect-tail-lash.json

```diff
@@ -26,6 +26,12 @@
                 "selector": "attack",
                 "type": "circumstance",
                 "value": -2
+            },
+            {
+                "key": "FlatModifier",
+                "selector": "skill-check",
+                "type": "circumstance",
+                "value": -2
```

Comment:
You're gonna need a Choice Set in this effect if you want to condense them. The penalty doesn't apply to both Strikes and skill checks. It applies to the type of roll that triggered it. Consider adding a `"removeAfterRoll": "if-enabled"` so the effect is removed after the triggering roll is done.

Reply:
It only applies to the triggering check anyways

Reply:
> It only applies to the triggering check anyways

@pedrogrullada is right its supposed to only effect the triggering roll, I did the suggested changes.